### PR TITLE
spidermonkey 38.2.1

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

File: Library/Formula/spidermonkey.rb [hunk 1]
```diff
@@ -37,13 +29,16 @@ def install
                                     "--with-nspr-prefix=#{Formula["nspr"].opt_prefix}",
                                     "--enable-macos-target=#{MacOS.version}"
 
-      inreplace "js-config", /JS_CONFIG_LIBS=.*?$/, "JS_CONFIG_LIBS=''"
+      cd "js/src" do
+        inreplace "js-config", /JS_CONFIG_LIBS=.*?$/, "JS_CONFIG_LIBS=''"
```
Comment (dunn) [inline Library/Formula/spidermonkey.rb]: Here you can just do ` inreplace "js/src/js-config"...`

**Comment by dunn** [comment]
Looks like CouchDB will need a `revision` increment.  Also
```
Error: 1 problem in 1 formula
spidermonkey:
 * Non-libraries were installed to "/usr/local/Cellar/spidermonkey/38.2.1/lib"
Installing non-libraries to "lib" is discouraged.
The offending files are:
  /usr/local/Cellar/spidermonkey/38.2.1/lib/libjs_static.ajs
```
Is that supposed to be `libjs_static.a`?

**Comment by dholm** [comment]
@dunn Looks like it is.  I'm investigating why it gets that weird name..

**Comment by apjanke** [comment]
SpiderMonkey 38.2.1 looks like a development release since it has "rc0" in its name. (It's weird that this is the version they have on their front page, though.) Should this go in a `devel` block instead of bumping the `stable` release?

I asked in [their newsgroup](https://groups.google.com/forum/#!searchin/mozilla.dev.tech.js-engine/rc0/mozilla.dev.tech.js-engine/1GC-7lkX-qk/Y4l3RpupAQAJ); let's see what they have to say.

**Comment by dholm** [comment]
@apjanke I've checked a few releases in the past and they all contained rcX in the filename so I sort of assumed that they simply didn't rename the one that was picked as stable.  Thanks for asking them about it!

**Comment by apjanke** [comment]
Okay, no problem about the release name.

This legacy-homebrew repo is deprecated now, though. Do you mind re-submitting this on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core) repo?

**Pull request closed without merging**